Cargo.toml

```diff
@@ -121,7 +121,7 @@ moka = { version = "0.12.10", default-features = false }
 multiversion = "0.8.0"
 num-traits = "0.2.19"
 num_enum = { version = "0.7.3", default-features = false }
-object_store = { version = "0.12.3", features = ["aws"] }
+object_store = { version = "0.12.3", default-features = false }
```

Comment:
where is this feature enabled now?

Reply:
I think this was enabled here for duckdb, so moved it there. Benchmarks and `vortex-ffi` already enabled a bunch of cloud variants.